src/coreclr/tools/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -3638,9 +3638,10 @@ void MethodContext::recGetThreadLocalStaticBlocksInfo(CORINFO_THREAD_STATIC_BLOC
 
     value.tlsIndex.handle                   = CastHandle(pInfo->tlsIndex.addr);
     value.tlsIndex.accessType               = pInfo->tlsIndex.accessType;
-    value.offsetOfMaxThreadStaticBlocks     = pInfo->offsetOfMaxThreadStaticBlocks;
-    value.offsetOfThreadLocalStoragePointer = pInfo->offsetOfThreadLocalStoragePointer;
-    value.offsetOfThreadStaticBlocks        = pInfo->offsetOfThreadStaticBlocks;
+    value.offsetOfMaxThreadStaticBlocks        = pInfo->offsetOfMaxThreadStaticBlocks;
```

Comment:
This function signature needs to match `getThreadLocalFieldInfo`: it needs to take a `bool isGCType`.

Then, instead of key `0` the key should be `0` for `isGCType == false` and `1` for `isGCType == true`.